### PR TITLE
Bump scala version

### PR DESCRIPTION
**Pull request by bkowalik**

### CODE REVIEW

File: src/main/scala/com/trueconnectivity/CommonConfigPlugin.scala [hunk 1]
```diff
@@ -71,10 +71,7 @@ object CommonConfigPlugin extends AutoPlugin {
         "-language:higherKinds",
         "-language:existentials",
         "-language:postfixOps",
-        "-Ywarn-adapted-args",
         "-Ywarn-dead-code",
-        "-Ywarn-adapted-args",
-        "-Ywarn-inaccessible",
```
Comment (bkowalik) [inline src/main/scala/com/trueconnectivity/CommonConfigPlugin.scala:77]: Those options are gone in Scala 2.13

**Comment by teroxik** [comment]
This makes sense. We have to plan how to make it work. Isn't it possible to override the version in the services? I mean we don't have to use the latest version of the plugin.

**Review by teroxik — APPROVED** [review]
Need to plan carefully this. - build on snapshots I would say.

**Comment by teroxik** [comment]
Not required anymore

**Pull request closed without merging**